backend/app/services/agent_orchestrator.py

```
@@ -116,13 +116,22 @@ def call(self, messages: List[Dict[str, str]], **kwargs) -> str:
         Returns:
             Generated response content
         """
-        # Convert to async call
-        loop = asyncio.new_event_loop()
-        asyncio.set_event_loop(loop)
-        try:
-            return loop.run_until_complete(self._async_call(messages, **kwargs))
-        finally:
-            loop.close()
+        import concurrent.futures
+        import threading
```

Ruff (0.12.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)